docs/Advanced-Concepts/Debugging-RTL.rst

```diff
@@ -86,9 +86,32 @@ identical program behavior. The torture utility can also be configured to run
 continuously for stress-testing. The torture utility exists within the ``utilities``
 directory.
 
+Dromajo Co-simulation for BOOM designs
```

Comment:
I don't think this should be under Debugging-RTL, since it doesn't support any type of customization.. I think it should be in a separate file (Debugging-BOOM, or Debugging-Cores)